internal/presenter/packages/spdx_json_presenter.go

```diff
@@ -67,7 +73,7 @@ func newSPDXJsonDocument(catalog *pkg.Catalog, srcMetadata source.Metadata) spdx
 			LicenseListVersion: spdxlicense.Version,
 		},
 		DataLicense:       "CC0-1.0",
-		DocumentNamespace: fmt.Sprintf("https://anchore.com/syft/image/%s", srcMetadata.ImageMetadata.UserInput),
+		DocumentNamespace: fmt.Sprintf("https://anchore.com/syft/image/%s", namespace),
```

Comment:
Now that the variable itself is called `namespace`, I think this would read better if we set the `DocumentNamespace` field to the value of `namespace` directly, so readers don't wonder about the two different possible meanings of "namespace".

Reply:
Makes sense - I can lift the `fmt.Sprintf` up so that we keep the naming similar.